cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -21,6 +21,13 @@ spec:
       containers:
       - name: ip-masq-agent
         image: gcr.io/google-containers/ip-masq-agent-amd64:v2.0.1
+        resources:
+          limits:
+            cpu: 10m
+            memory: 8Mi
```

Comment:
And if we're concerned about  memory, we should set `GOGC=20` or something.
https://golang.org/pkg/runtime/